Cloudformation helper fails to update when using an S3 url

When you call the `create_or_update` method and pass in a `TemplateUrl` rather than a `TemplateBody`, an exception is raised due to the changeset name being created as the hash of the template body, e.g.:

```python
with XAccountClient("cloudformation", role_arn, "session name") as deployment_account_cloudformation:
    deployment_account_cloudformation.create_or_update(
        Capabilities="CAPABILITY_NAMED_IAM",
        Parameters=STACK_PARAMETERS,
        StackName=STACK_NAME,
        TemplateURL=template_url
    )
```
results in:
```
[ERROR] TypeError: descriptor 'encode' for 'str' objects doesn't apply to a 'NoneType' object
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 34, in lambda_handler
    update_stack(record)
  File "/var/task/lambda_function.py", line 51, in update_stack
    deployment_account_cloudformation.create_or_update(
  File "/opt/python/betterboto/cloudformation.py", line 56, in create_or_update
    change_set_name = get_hash_for_template(kwargs.get('TemplateBody'))
  File "/opt/python/betterboto/cloudformation.py", line 13, in get_hash_for_template
    hasher.update(str.encode(template))
```

--- a/betterboto/cloudformation.py
+++ b/betterboto/cloudformation.py
@@ -53,7 +53,7 @@ def create_or_update(self, **kwargs):
             raise e
     else:
         logger.info('Updating: {}'.format(stack_name))
-        change_set_name = get_hash_for_template(kwargs.get('TemplateBody'))
+        change_set_name = get_hash_for_template(kwargs.get('TemplateBody', kwargs.get('TemplateURL')))
         self.create_change_set(
             ChangeSetName=change_set_name,
             ChangeSetType="UPDATE",

